### ISSUE TITLE
Parametric types

### ISSUE THREAD

**Issue by jkrumbiegel** [JuliaInterop/ObjectiveC.jl#19]
In my ScriptingBridge project, some properties are `SBElementArray`s where the element type is known. I don't think there's a way right now to map that, correct? At least variations of `@objcwrapper MyArray{T<:Integer} <: NSArray` don't work. Would be nice to have something like that, because currently I have to manually reinterpret the `id{NSObject}`s to what I know they really are.

**Comment by maleadt** [comment]
Yeah, that's not supported now. The NSArray wrapper also suffers from this, hence it returning an `id` too.
